Sources/Plasma/PubUtilLib/plMessage/plVaultNotifyMsg.h

```diff
@@ -75,8 +76,8 @@ class plVaultNotifyMsg : public plMessage
     uint16_t  GetType() const { return fType; }
     void      SetType( uint16_t v ) { fType=v; }
 
-    int8_t  GetResultCode() const { return fResultCode; }
-    void    SetResultCode( int8_t v ) { fResultCode=v; }
+    ENetError GetResultCode() const { return fResultCode; }
+    void SetResultCode(ENetError resultCode) { fResultCode = resultCode; }
```

Comment:
This message probably predates `ENetError`, but I don't think it's ever serialized, so as long as we internally agree on what the error codes mean, everything will be fine. This PR moves us toward more consistency, so this is probably for the best.

Reply:
Yeah, I didn't see any place where `plVaultNotifyMsg` gets sent over the network (and I assume it shouldn't appear in any PRPs...). At least one place already stored an `ENetError` value in this field anyway, so it seemed like a logical choice.